dockerbuild/wait_for_mongodb.sh

```diff
@@ -14,14 +14,12 @@ until mongo "$host"/foo --eval "db.getCollectionNames()" > /dev/null 2>&1; do
     >&2 echo "MongoDB is unavailable - sleeping"
     sleep 1
 done
-
 >&2 echo "MongoDB is up - executing command"
 
+
 # Substitute the desired MongoDB hostname in the built project's conf file.
 before="$PWD"
```

Comment:
You can remove line 21 & 23 (they don't do anything now).

Reply:
done